### ISSUE TITLE
[BUG] Need a better tutorial for new users

### ISSUE THREAD

**Issue by farling42** [p4535992/foundryvtt-lights-hud-ate#1]
**Module Version:** v0.11.8

**Describe the bug**
The three icons that appeared on the token with the original Lights HUD module are not appearing when I right-click on a token with your forked version.

**To Reproduce**
Right-click on a PCs token, optionally with torches in their inventory, or the light spell available.

**Expected behavior**
Three icons should appear just like displayed in the image on https://github.com/Malekal4699/LightsHUD

**Screenshots**

**Browser:**
 Chrome Version 101.0.4951.54 (Official Build) (64-bit)

**Foundry Version:** 
Foundry VTT 9.269

**Game System:** 
DND5E

**Additional context**
If there is "more power" in the new module, it would still be useful for novices to see the existing three icons automatically added to tokens.

**Comment by farling42** [comment]
I should add that instead of the three icons, I see a single icon with a light bulb on.
Clicking on the light bulb I see two options:
- [GM] Call macro preset
- [GM] Call macro custom

**Comment by farling42** [comment]
But your module is trying to replace Lights, which didn't need any knowledge of ATE, and the original Lights provided buttons out-of-the box.
Unfortunately, the additions in your version of the module have added complexity for people who have no previous knowledge of effects.
Perhaps I should just fork the original simple version and keep it active for folks who don't have the time/knowledge to handle lots of effects?

**Comment by p4535992** [comment]
Well the project was born as a more customizable variant of LigthHUD for people a little more "experienced", but have I only discovered now that the LightHUD developer has put his project on standby? (this explains the download boom), I want to try to do something within everyone's reach ... but it will take me some time to dedicate ... i was thinking to do something like i have done [here](https://github.com/p4535992/environment-interaction-multisystem/blob/main/wiki/how_to_use.md) .

Basically I put a configuration dialog button on the item to emulate ATE's work without touch any active effects. It should emualte the old LIghtHUD, but still more dynamic because every item can be customizable on is own way, or at least give a choice option for those who want to use or not use ATE ....

What did you think ?

**Comment by farling42** [comment]
Yes, the download boom is cause the author of LightHUD has marked your module as the replacement.

The old LightHUD just worked by adding those buttons without requiring any user configuration at all - it simply works.

**Comment by p4535992** [comment]
So I was unable to integrate the solution with "zero configuration" because it would have made me lose the dynamism of the customization of the lights.
I applied a solution without active effect because it was also requested by others, but you still have to do some minimal configuration:

- Enable the module settings "Apply HUD interface on flagged Items" and disable "Apply HUD interface on ATE Items"
- Edit the item you want to give off light or vision
- Click on the form button
- Edit the form as you like either with a system preset or with a completely customized one
- Close the form.
- The item is now viewable on the HUD interface
- More details on these features on the readme 

Here a video:

![img2](https://github.com/p4535992/foundryvtt-lights-hud-ate/raw/master/wiki/assets/light_hud_flags.gif)
